lib/command-logger.coffee

```diff
@@ -162,6 +162,7 @@ class CommandLogger
   #
   # Returns the {String} format of the command time.
   formatTime: (time) ->
+    moment ?= require 'moment'
```

Comment:
It's only called when the commands are formatted for a report. So this is safe I think.